packages/integrations/vercel/test/isr.test.js

```diff
@@ -33,10 +33,18 @@ describe('ISR', () => {
 				src: '^/two$',
 				dest: '_render',
 			},
+			{
+				src: '^\\/excluded\\/([^/]+?)$',
```

Comment:
I noticed that the `src` fields created did not escape `/`, unlike core. The escaping may not be necessary, but it's worth confirming. Do you know if vercel handles this correctly unescaped?